frontend/src/components/CodingRoomComponents/QuestionDrawer.js

```diff
@@ -45,7 +45,26 @@ function QuestionDrawer (props) {
     setQuestionTitle(data.questionTitle)
     setDifficulty(data.difficulty)
     setQuestionDescription(data.questionDescription)
-    setQuestionExamples(data.examples)
+
+    const examples = data.examples
+
+    if (examples.constructor.name !== 'Array') {
```

Comment:
I think this is fine as a stop gap solution, but the data from the backend should be standardised as either array or string.